docs/data/charts/lines/lines.md

```diff
@@ -175,6 +175,16 @@ Different series could even have different interpolations.
 
 {{"demo": "InterpolationDemoNoSnap.js", "hideToolbar": true}}
 
+### Baseline
+
+The area chart draws a `baseline` on the Y axis `0`.
+This is useful as a base value, but customized visualizations may require a different baseline.
+
+To change it, simply provide the wanted `baseline` number of a series.
+Different series can have different `baselines`.
```

Comment:
It's already not recommended to have multiple areas on a charts (except if staking them) let's not encourage use to have multiple areas with different baselines 😅
```suggestion
```